when I add  s.dependency "Firebase/Core","5.0.0" in Podspec,then Use pod package failed

## I want to make private pod and i depend firebase， I add s.dependency "Firebase/Core","5.0.0" in Podspec, then I want to make a framework , I use pod package , then build failed. 

 /var/folders/qc/7_dh4q2x5_9g4ydsyn4_r03h0000gn/T/cocoapods-xgidm4bd/Pods/Static/FirebaseCore/Firebase/Core/FIRVersion.m:18:2: error: "Firebase_VERSION is not defined: add -DFirebase_VERSION=... to the build invocation"
    #error "Firebase_VERSION is not defined: add -DFirebase_VERSION=... to the build invocation"
     ^
    /var/folders/qc/7_dh4q2x5_9g4ydsyn4_r03h0000gn/T/cocoapods-xgidm4bd/Pods/Static/FirebaseCore/Firebase/Core/FIRVersion.m:22:2: error: "FIRCore_VERSION is not defined: add -DFIRCore_VERSION=... to the build invocation"
    #error "FIRCore_VERSION is not defined: add -DFIRCore_VERSION=... to the build invocation"
     ^
    2 errors generated.
    
    ** BUILD FAILED **
    
    
    The following build commands failed:
    	CompileC /var/folders/qc/7_dh4q2x5_9g4ydsyn4_r03h0000gn/T/cocoapods-xgidm4bd/Pods/build/Pods.build/Release-iphoneos/FirebaseCore.build/Objects-normal/arm64/FIRVersion.o FirebaseCore/Firebase/Core/FIRVersion.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    (1 failure)



How can I solve It ?

--- a/FirebaseCore.podspec
+++ b/FirebaseCore.podspec
@@ -33,7 +33,8 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   ]
   s.dependency 'GoogleToolboxForMac/NSData+zlib', '~> 2.1'
   s.pod_target_xcconfig = {
-    'OTHER_CFLAGS' => '-fno-autolink ' +
-      '-DFIRCore_VERSION=' + s.version.to_s + ' -DFirebase_VERSION=5.1.0'
+    'OTHER_CFLAGS' => '-fno-autolink',
+    'GCC_PREPROCESSOR_DEFINITIONS' =>
+      'FIRCore_VERSION=' + s.version.to_s + ' Firebase_VERSION=5.1.0'
   }
 end

--- a/FirebaseFirestore.podspec
+++ b/FirebaseFirestore.podspec
@@ -61,7 +61,8 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
       '"${PODS_TARGET_SRCROOT}/Firestore/third_party/abseil-cpp" ' +
       '"${PODS_ROOT}/nanopb" ' +
       '"${PODS_TARGET_SRCROOT}/Firestore/Protos/nanopb"',
-    'OTHER_CFLAGS' => '-DFIRFirestore_VERSION=' + s.version.to_s + ' -DPB_FIELD_16BIT'
+    'GCC_PREPROCESSOR_DEFINITIONS' =>
+      'FIRFirestore_VERSION=' + s.version.to_s + ' PB_FIELD_16BIT'
   }
 
   s.prepare_command = <<-CMD

--- a/FirebaseFirestore.podspec
+++ b/FirebaseFirestore.podspec
@@ -55,14 +55,13 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
   s.frameworks = 'MobileCoreServices'
   s.library = 'c++'
   s.pod_target_xcconfig = {
-    'GCC_PREPROCESSOR_DEFINITIONS' => 'GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 ',
+    'GCC_PREPROCESSOR_DEFINITIONS' => 'GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 ' +
+      'FIRFirestore_VERSION=' + s.version.to_s + ' PB_FIELD_16BIT',
     'HEADER_SEARCH_PATHS' =>
       '"${PODS_TARGET_SRCROOT}" ' +
       '"${PODS_TARGET_SRCROOT}/Firestore/third_party/abseil-cpp" ' +
       '"${PODS_ROOT}/nanopb" ' +
-      '"${PODS_TARGET_SRCROOT}/Firestore/Protos/nanopb"',
-    'GCC_PREPROCESSOR_DEFINITIONS' =>
-      'FIRFirestore_VERSION=' + s.version.to_s + ' PB_FIELD_16BIT'
+      '"${PODS_TARGET_SRCROOT}/Firestore/Protos/nanopb"'
   }
 
   s.prepare_command = <<-CMD

